Makefile

```diff
@@ -9,10 +9,13 @@ VERSION_DATE := $(shell $(MAKEFILE_PATH)/commit_date.sh)
 default: truss
 
 dependencies:
+ifeq ($(GO111MODULE), on)
+	go mod download
+endif
```

Comment:
Shouldn't need this block since the below block will download the deps for you and install them.

Reply:
Ah ok cool, I'd prefer to avoid go get entirely if all the other deps in the go.mod file are direct deps of the one's below. I'll work on getting these in there so `go mod download` is all that is needed.

Reply:
The problem is, the below deps are not merely imports used in truss itself, but binaries that truss calls out to. The goal of `make dependencies` is to get those executables in your PATH. I don't believe `go mod download` nor enforcing them in the `go.mod` will function. The CI is a good test if this however, so if you see it working there you're probably in the clear.

Reply:
Ah I see, yeah I think you are right. That is kind of frustrating that they won't do installs from modules as well as downloading them considering it's meant to be the main dependency management option going forward. Hm might still be a place for custom tools to wrap modules and handle this all properly.